Makefile

```
@@ -21,4 +21,4 @@ fmt:
 
 .PHONY: test
 test:
-	go test -race -v ./...
+	go run gotest.tools/gotestsum --format testname -- -race -v ./...
```

it does not generate JUnit report, Is it intended? if not

```suggestion
	go run gotest.tools/gotestsum --format testname --junitfile junit-report.xml -- -race -v ./...
```